website/content/community/maintainers.md

```diff
@@ -41,16 +41,26 @@ maintainer should object to adding the new maintainer.
 
 When adding a new maintainer, we should:
 
-* Add to CODEOWNERS in the metallb repo.
+* Add to CODEOWNERS in the metallb repo and to the MetalLB github org.
```

Comment:
Can you also mention the metallb-operator ? I think it's mature enough and maintainers should have access there

Reply:
Sure!